qc_tests/streaks.py

```diff
@@ -203,29 +532,50 @@ def rsc(station, var_list, config_dict, full=False, plots=False, diagnostics=Fal
     for var in var_list:
 
         obs_var = getattr(station, var)
+        if obs_var.name == "wind_direction":
+            wind_speed = copy.deepcopy(getattr(station, "wind_speed"))
+        else:
+            wind_speed = None
 
         # need to have at least two observations to get a streak
         if len(obs_var.data.compressed()) >= 2:
-
-            # repeating (straight) strings
             if full:
-                get_repeating_string_threshold(obs_var, config_dict, plots=plots, diagnostics=diagnostics)
```

Comment:
Similar terminology to streak testing. Is it using same function or could it be uniquely named?